### PR TITLE
enable abn for istio 1.7

### PR DESCRIPTION
**Pull request by kalantar**

### CODE REVIEW

**Comment by fabolive** [comment]
@kalantar: There seems to have been changes in how to use `istioctl manifest` and `istioctl install` in newer versions of Istio. It seems to me that the command below should work for all versions:

```bash
istio-${ISTIO_VERSION}/bin/istioctl manifest generate  --set profile=demo \
--set values.kiali.enabled=false --set values.prometheus.enabled=true \
| kubectl apply -f -
```

In particular, you will need replace the line below with the one above:

https://github.com/iter8-tools/iter8/blob/801158acfb478339be0ef351999a7c723a2b3bdc/test/e2e/install-istio.sh#L12-L13

**Comment by fabolive** [comment]
I will take my comment above back. My suggestion above might lead the installation process to get stuck in Istio 1.7. I suggest the following, instead For Istio 1.7 and beyond, we will need the command below:

```
istio-${ISTIO_VERSION}/bin/istioctl manifest install --set profile=demo \ 
 --set values.kiali.enabled=false --set values.grafana.enabled=false 
```
Note that the change is that the `manifest` command now has a `install` subcommand, which used to be `apply`

**Review by fabolive — DISMISSED** [review]
This looks good to me, but I do have a question below, @kalantar.

**Comment by fabolive** [comment]
@kalantar: Did you forget to re-generate the default deployment files after your last commit?

**Comment by kalantar** [comment]
> @kalantar: Did you forget to re-generate the default deployment files after your last commit?

Yes, apparently. Done.

File: install/iter8-controller.yaml [hunk 1]
```diff
@@ -21,8 +27,6 @@ data:
       query_template: sum(increase(istio_requests_total{reporter='source',job='istio-mesh'}[$interval])) by ($version_labels)
     - name: iter8_total_latency
       query_template: (sum(increase(istio_request_duration_seconds_sum{reporter='source',job='istio-mesh'}[$interval])) by ($version_labels))*1000
-      units: msec # optional
```
[the remark this answers is not in the document]
Comment (kalantar) [reply install/iter8-controller.yaml:24]: done

**Review by fabolive — APPROVED** [review]
LGTM

**Pull request merged**